rspirv/sr/autogen_ops.rs

```diff
@@ -47,13 +47,20 @@ pub enum Branch {
         size: u32,
     },
     TerminateInvocation,
+    DemoteToHelperInvocation,
```

Comment:
Perhaps this belongs with one of the block terminators in `rspirv/grammar/reflect.rs`.

Reply:
Based on the specification wording I would exclude it
>Unlike the [OpTerminateInvocation](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#OpTerminateInvocation) instruction, this does not necessarily terminate the invocation which might be needed for derivative calculations. It is not considered a flow control instruction (flow control does not become non-uniform) and does not terminate the block.

Reply:
Oh cool, thanks! I had a hard time finding the certain ops such as `EmitMeshTasksEXT`, `SetMeshOutputsEXT`, `HitObject*NV`, `ReorderThead*NV` andsoforth in [the spec](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html) but at least this one is properly documented.